### DISCUSSION TITLE
controller brick ?

### DISCUSSION THREAD

**Discussion by lucaslngls** [danielnilsson9/bbs-fw#234] [Q&A]
Hi guys,
I flashed my bbs02b a few weeks ago.
Today I went back into the software to tweak my settings, I regularly saved my changes until at one point an error message appeared saying something like controller not found. Since then the bike does not turn on anymore. I want to point out that just before that while the controller was connected to the computer, I went into my battery application to check its status visually, moreover the last change made was the start assistance level.
So I tried to reflash the controller but the STC-ISP software did not even manage to check the mcu, I think it can not detect it.

Do you have any leads?
Thanks for your help

**Comment by danielnilsson9** [comment]
Unlikely that you controller was bricked by software. It is more likely you cable was damaged if you have been unplugging and replugging it when changing settings, unfortunately quite common... Happened to me too, can usually be fixed and resoldered but of course a bit of work.
